packages/language/src/languages.json

```diff
@@ -1 +1 @@
e","heb","Hebrew"],["hz","her","Herero"],["hi","hin","Hindi"],["ho","hmo","Hiri Motu"],["hr","hrv","Croatian"],["hu","hun","Hungarian"],["hy","hye","Armenian"],["ig","ibo","Igbo"],["io","ido","Ido"],["ii","iii","Sichuan Yi"],["iu","iku","Inuktitut"],["ie","ile","Interlingue"],["ia","ina","Interlingua"],["id","ind","Indonesian"],["ik","ipk","Inupiaq"],["is","isl","Icelandic"],["it","ita","Italian"],["jv","jav","Javanese"],["ja","jpn","Japanese"],["kl","kal","Kalaallisut"],["kn","kan","Kannada"],["ks","kas","Kashmiri"],["ka","kat","Georgian"],["kr","kau","Kanuri"],["kk","kaz","Kazakh"],["km","khm","Central Khmer"],["ki","kik","Kikuyu"],["rw","kin","Kinyarwanda"],["ky","kir","Kirghiz"],["kv","kom","Komi"],["kg","kon","Kongo"],["ko","kor","Korean"],["kj","kua","Kuanyama"],["ku","kur","Kurdish"],["lo","lao","Lao"],["la","lat","Latin"],["lv","lav","Latvian"],["li","lim","Limburgan"],["ln","lin","Lingala"],["lt","lit","Lithuanian"],["lb","ltz","Luxembourgish"],["lu","lub","Luba-Katanga"],["lg","lug","Ganda"],["mh","mah","Marshallese"],["ml","mal","Malayalam"],["mr","mar","Marathi"],["mk","mkd","Macedonian"],["mg","mlg","Malagasy"],["mt","mlt","Maltese"],["mn","mon","Mongolian"],["mi","mri","Maori"],["ms","msa","Malay"],["my","mya","Burmese"],["na","nau","Nauru"],["nv","nav","Navajo"],["nr","nbl","Ndebele, South"],["nd","nde","Ndebele, North"],["ng","ndo","Ndonga"],["ne","nep","Nepali"],["nl","nld","Dutch"],["nn","nno","Norwegian Nynorsk"],["nb","nob","Bokmål, Norwegian"],["no","nor","Norwegian"],["ny","nya","Chichewa"],["oc","oci","Occitan"],["oj","oji","Ojibwa"],["or","ori","Oriya"],["om","orm","Oromo"],["os","oss","Ossetian"],["pa","pan","Panjabi"],["pi","pli","Pali"],["pl","pol","Polish"],["pt","por","Portuguese"],["br","por","Brazilian Portuguese"],["ps","pus","Pushto"],["qu","que","Quechua"],["rm","roh","Romansh"],["ro","ron","Romanian"],["rn","run","Rundi"],["ru","rus","Russian"],["sg","sag","Sango"],["sa","san","Sanskrit"],["si","sin","Sinhala"],["sk","slk","Slovak"],["sl","slv","Slovenian"],["se","sme","Northern Sami"],["sm","smo","Samoan"],["sn","sna","Shona"],["sd","snd","Sindhi"],["so","som","Somali"],["st","sot","Sotho, Southern"],["es","spa","Spanish"],["sq","sqi","Albanian"],["sc","srd","Sardinian"],["sr","srp","Serbian"],["ss","ssw","Swati"],["su","sun","Sundanese"],["sw","swa","Swahili"],["sv","swe","Swedish"],["ty","tah","Tahitian"],["ta","tam","Tamil"],["tt","tat","Tatar"],["te","tel","Telugu"],["tg","tgk","Tajik"],["tl","tgl","Tagalog"],["th","tha","Thai"],["ti","tir","Tigrinya"],["to","ton","Tonga"],["tn","tsn","Tswana"],["ts","tso","Tsonga"],["tk","tuk","Turkmen"],["tr","tur","Turkish"],["tw","twi","Twi"],["ug","uig","Uighur"],["uk","ukr","Ukrainian"],["ur","urd","Urdu"],["uz","uzb","Uzbek"],["ve","ven","Venda"],["vi","vie","Vietnamese"],["vo","vol","Volapük"],["wa","wln","Walloon"],["wo","wol","Wolof"],["xh","xho","Xhosa"],["yi","yid","Yiddish"],["yo","yor","Yoruba"],["za","zha","Zhuang"],["zh","cmn","Chinese"],["zu","zul","Zulu"]]
```

Comment:
I see here 
```json
["br","bre","Breton"]
```
and
```json
["br","por","Brazilian Portuguese"]
```
this can drive to a collision in cases like       
```js
this.languagesAlpha2[language.alpha2] = language;
```

Reply:
I understand, but we have a small problem, in this case you use ISO 639-2 and I couldn't find Brazilian Portuguese in this ISO.

My suggestion would be to change from:
["br","por","Brazilian Portuguese"]

For:
["pt-BR","por","Brazilian Portuguese"]

In this case, we would use the IETF https://en.wikipedia.org/wiki/IETF_language_tag.